### ISSUE TITLE
Constant error when opening the mintty console

### ISSUE THREAD

**Issue by BlackScreen** [mintty/mintty#669]
For a few weeks I am getting the following error when opening the mintty console:
```
Font not found, using system substitute:
Consolas
```
I am using the media-autobuild_suite and the error did not appear before with same settings.
Is there anything I can do to fix it? Or can you fix this? Thanks!

**Comment by mintty** [comment]
Open the Options dialog and choose your own font, so it will be saved to your mintty config file instead of Consolas.

**Comment by BlackScreen** [comment]
That did fix the problem. Thanks.

**Comment by mintty** [comment]
What is actually the media-autobuild suite? Is it something that comes with mintty embedded, and did it perhaps preconfigure mintty for Consolas?

**Comment by BlackScreen** [comment]
It's a suite that uses mintty to build i.e. FFmpeg - see here: https://github.com/jb-alvarado/media-autobuild_suite.
It is preconfigured with Consolas. It always was. Then the problems occured (after a update in mintty), then the author changed to config, now he changed it back. Probably something in mintty did change. But the error is now gone for me. There still seems to be something wrong with handling UTF-8 characters.
See here - copied out of mintty from the output of the suite:
```
  Running git clone for speex...
â”Œ speex git ......................................... [Recently updated]
â”œ Running autoreconf...
â”œ Running configure...
â”œ Running make...
â”œ Stripping...
â”” speex-git .................................................. [Updated]
```
This should be vertical lines.

**Comment by mintty** [comment]
Consolas is actually included by default since [Windows 7](https://www.microsoft.com/typography/fonts/product.aspx?PID=161). Did you explicitly remove it or do you still run Windows XP?

**Comment by mintty** [comment]
About UTF-8, how is mintty configured in that suite? What's the output of `locale`, `echo $LC_ALL`, `echo $LC_CTYPE`, `echo $LANG`? What does the Options menu, section Text, say about Character set?

**Comment by BlackScreen** [comment]
Consolas is installed on my system. I have Windows 7, 64 Bit.
```
$ locale
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_ALL=
$ echo $LC_ALL

$ echo $LC_TYPE

$ echo $LANG
en_US.UTF-8
```
The options menu also says "UTF-8". Ideas?

**Comment by mintty** [comment]
About the font, maybe your config file has a CRLF (DOS/Windows) line end there, so the font name does not match. This strictness is released again with mintty 2.7.8, I suppose your version is older.
About UTF-8, is the program (git?) a Windows build (rather than cygwin build)? Did it work in an older version? Apart from these hints I cannot really analyse it remotely.

**Comment by BlackScreen** [comment]
You are right. I am using version 2.7.7 - I guess the packet manager is not updated yet to use the newer version. Thank you very much for the tip with CRLF - I replaced all ocurences of CRLF with Notepad++ to only leave LF and now everything is working again. Thanks!
